src/content/advanced/cluster-management/node-pools-capi/index.md

```diff
@@ -0,0 +1,322 @@
+---
+linkTitle: Node pools (CAPI)
+title: Node pools (CAPI)
+description: A general description of node pools as a concept, its benefits, and some details you should be aware of.
+weight: 50
+menu:
+  main:
+    parent: advanced-cluster-management
+user_questions:
+  - What is a node pool?
+  - What are node pools?
+  - In which cloud environments are node pools supported?
+  - Which workload cluster releases introduced node pools?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+last_review_date: 2023-11-22
+---
+
+## Definition
+
+A node pool is a set of nodes within a Kubernetes cluster that share the same configuration (machine type, CIDR range, etc.). Each node in the pool is labeled by the node pool's name.
+
+## Advantages
+
+Prior to the introduction of node pools, a cluster could only comprise one type of worker node. The cluster would have
+to be scaled as a whole, and the availability zone distribution would apply to all worker nodes of a cluster. This
+would mean that every worker node would have to be big enough to run the largest possible workload, in terms of
+memory and CPU resources required. At the same time, all worker nodes in the cluster would have to use the same
+availability zone distribution, even if some workloads wouldn't require the increased availability.
+
+Node pools are independent groups of worker nodes belonging to a cluster, where all nodes within a pool share a
+common configuration. You can combine any type of node pool within one cluster. Node pools can differ regarding:
+
+- Machine type
+- Availability zone distribution
+- Scaling configuration (number of nodes)
+
+A node pool is identified by a name that you can pick as a cluster administrator. The name of the node pool is set as a label on all nodes belonging to the pool.
```

Comment:
This is a good place to mention limitation of the nodepool name, like how many chars and waht chars are allowed + few examples

Reply:
And let's also be specific. The introduction already talked about name labeling. Let's reference the actual `giantswarm.io/machine-pool=...` label here. Or since you already specify it below in the per-provider section, maybe rather remove this sentence here (see also suggestion above to add it as bullet point).